ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/viewmodel/CustomerCenterViewModel.kt

```diff
@@ -341,38 +401,60 @@ internal class CustomerCenterViewModelImpl(
         }
     }
 
-    private suspend fun loadPurchaseInformation(
+    private fun computeMainScreenPaths(state: CustomerCenterState.Success): List<CustomerCenterConfigData.HelpPath> {
+        val managementScreen = state.customerCenterConfigData.getManagementScreen()
+        val baseSupportedPaths = managementScreen?.let { screen ->
+            val selectedPurchase = if (state.purchases.size == 1) {
+                state.purchases.first()
+            } else {
+                null
+            }
+            supportedPaths(selectedPurchase, screen)
+        } ?: emptyList()
+
+        // For main screen: if multiple purchases, show only general paths
+        // If single purchase or no purchases, show all available paths
+        return if (state.purchases.size > 1) {
+            PathUtils.filterGeneralPaths(baseSupportedPaths)
+        } else {
+            baseSupportedPaths
+        }
+    }
+
+    private suspend fun loadPurchases(
         dateFormatter: DateFormatter,
         locale: Locale,
-    ): PurchaseInformation? {
+    ): List<PurchaseInformation> {
         val customerInfo = purchases.awaitCustomerInfo(fetchPolicy = CacheFetchPolicy.FETCH_CURRENT)
 
         val hasActiveSubscriptions = customerInfo.activeSubscriptions.isNotEmpty()
         val hasNonSubscriptionTransactions = customerInfo.nonSubscriptionTransactions.isNotEmpty()
 
         if (hasActiveSubscriptions || hasNonSubscriptionTransactions) {
-            val activeTransactionDetails = findActiveTransaction(customerInfo)
-
-            if (activeTransactionDetails != null) {
-                val entitlement = customerInfo.entitlements.all.values
-                    .firstOrNull { it.productIdentifier == activeTransactionDetails.productIdentifier }
-
-                return createPurchaseInformation(
-                    activeTransactionDetails,
-                    entitlement,
-                    customerInfo.managementURL,
-                    dateFormatter,
-                    locale,
-                )
+            val activeTransactions = findActiveTransactions(customerInfo)
```

Comment:
nit: there's no longer an `active` transaction, right? In iOS we're calling them:
```swift
    var subscriptionsSection: [PurchaseInformation] = []
    var nonSubscriptionsSection: [PurchaseInformation] = []
```